cubi_isa_templates/isatab-sc_rnaseq-multiplex/{{cookiecutter.__output_dir}}/i_Investigation.txt

```diff
@@ -0,0 +1,99 @@
+ONTOLOGY SOURCE REFERENCE
+Term Source Name	UO	OBI	NCBITAXON	OMIM	HP	ORPHA
+Term Source File	http://data.bioontology.org/ontologies/UO	http://data.bioontology.org/ontologies/OBI	http://data.bioontology.org/ontologies/NCBITAXON	http://data.bioontology.org/ontologies/OMIM	http://data.bioontology.org/ontologies/HP	http://data.bioontology.org/ontologies/ORDO
+Term Source Version	48	35	10	12	570	2.8
```

Comment:
There are newer versions of these ontologies available. We don't use this information though, so probably irrelevant?